diff --git a/lib/Minify/HTML.php b/lib/Minify/HTML.php
index 0771c3b..4de7f29 100644
--- a/lib/Minify/HTML.php
+++ b/lib/Minify/HTML.php
@@ -152,7 +152,6 @@ class Minify_HTML
         );
 
         // use newlines before 1st attribute in open tags (to limit line lengths)
-        $this->_html = preg_replace('/(<[a-z\\-]+)\\s+([^>]+>)/iu', "$1\n$2", $this->_html);
 
         // fill placeholders
         $this->_html = str_replace(
